### ISSUE TITLE
Elasticsearch-mesos ignores scale up request after adding more mesos worker nodes.

### ISSUE THREAD

**Issue by zoza1982** [mesos/elasticsearch#520]
I think i found bug in ELK framework ( framework related ). If I have existing mesos and I install elasticsearch-mesos framework...I can scale up and down based on my max number of mesos worker nodes. If I then ....just add more mesos worker nodes into existing cluster ( and not touching ES framework ) and try to expand Elasticsearch to to new number of worker nodes....it does nothing. Simply ignores me. It seems like ELK bounds to original number of worker nodes when it got initially installed. Tested it several times... by also rebuilding Mesos cluster from scratch..same results.

I will post mesos logs here when I collect....( having trouble finding them...??? ) just nothing there...

**Comment by zoza1982** [comment]
I searched elasticsearch both stdour sterr sanbox logs and I find nothing related to scaling...no resource offers or anything like it. I also looked at messos logs and same thing.

I also noticed, that my scaling down ( within the original number of worker nodes ) now also stopped working. ES UI will get updated...but I am not seeing tasks going down to new requested number anymore.

**Comment by zoza1982** [comment]
I run 0.7.1 framework by the way. Forgot to mention.

**Comment by philwinder** [comment]
Hi @zoza1982 That should definitely work. I will try and replicate myself. Thanks.

**Comment by zoza1982** [comment]
@philwinder please let me know if you can re-produce it by following exact steps I described.

**Comment by philwinder** [comment]
@zoza1982 I've just tried to recreate your issue, but it works fine for me. Here is what I did (you're instructions are a little hard to follow):
1. Used https://github.com/philwinder/mesos-terraform to create a cluster of three nodes on AWS.
2. Installed ES as instructed. Three elasticsearch tasks were created.
3. Edited the terraform scripts (changed the number of slaves to 4) to add another slave.
4. Scale the framework to 4 using the GUI. Another task was created.
5. Scale the framework to 1 using the GUI. All tasks but one were destroyed.

This is working as expected.

I will close this now. If you continue to have problems, please put your setup in a script to make it easy for me to test.

Thanks, Phil.

**Comment by zoza1982** [comment]
@philwinder thanks for testing...could you let me know in your test did you run framework in Jar or container mode? and which version of the framework?

**Comment by philwinder** [comment]
Container mode, with 1.0.0. (The scaling functionality hasn't changed).

**Comment by zoza1982** [comment]
Thanks. We are here 0.7.1 jar mode for Mantl.io
